packages/ui/src/components/ProTable/Pagination/index.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { ReactElement } from 'react';
+import { DoubleLeftOutlined, LeftOutlined, RightOutlined } from '@ant-design/icons';
+import { Button, Select, Space } from 'antd';
+
+import { IProTablePagination } from '../types';
+import { reverseSortDirection } from '../utils';
+
+import { PaginationViewPerQuery } from './constants';
+import { getPaginationOptions } from './utils';
+
+import styles from '@ferlab/style/components/protable/Pagination.module.scss';
+
+const iconSize = 11;
+
+const Pagination = ({
+    current,
+    defaultViewPerQuery,
+    dictionary,
+    onChange,
+    onPageChange,
+    onShowSizeChange,
+    queryConfig,
+    searchAfter,
+    setQueryConfig,
+    total,
+}: IProTablePagination): ReactElement => {
```

Comment:
`IProTablePagination` pourrait etre `IPaginationProps` et surtout etre posé juste au dessus de `const Pagination`: ca augmente la vitesse de lecture des types des props du component, sans devoir aller dans un autre fichier les chercher. Quand tu debug/parcours toute la codebase a coup de component to component c'est un bien fait considérable.

Reply:
Je vais renommer pour IPaginationProps qui s'applique mieux. Je suis assi d'accord avec toi que qu'on devrait mettre certaines interface dans le fichier même sauf que dans notre cas, on divise déjà `type.ts`par défaut (qui lui aussi utilise IPaginationProps). Donc  pour conserver ce qu'on fait déjà je vais le conserver dans `type.ts`. Pour la lecture rapide, on peut aussi faire ctrl+click sur la propriété pour voir sa défini dans vscode même si ce n'est pas l'idéal.